virttest/virt_vm.py

```diff
@@ -1207,6 +1207,11 @@ def print_guest_network_info():
         # try to login if VM bootup really, at least once
         not_tried = True
         end_time = start_time + timeout
+        logger = None
+        if self.params.get("skip_non_error_waiting_for_login", "no") == "yes":
```

Comment:
Well this could lead to very hard to debug issues especially. If you really want to eliminate the duplicate logs, let's just keep a `set` of `str(err)`s (or at least the last one) and only report it for the first time (or on change).